samples/appending-events/Program.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using EventStore.Client;
+
+namespace appending_events {
+	class Program {
+		static void Main(string[] args) {
+			var settings = new EventStoreClientSettings {
```

Comment:
should we still be bypassing this?

Reply:
I think that it should be fine to use connection string for now. I can try to update that.

Reply:
Updated in https://github.com/EventStore/EventStore-Client-Dotnet/pull/101/commits/61a182eb3f1b4c276d4f48f277ddac8bf26a45e4.